esrally/client.py

```diff
@@ -12,7 +12,12 @@ class EsClientFactory:
     Abstracts how the Elasticsearch client is created. Intended for testing.
     """
     def __init__(self, hosts, client_options):
-        logger.info("Creating ES client connected to %s with options [%s]" % (hosts, client_options))
+        masked_client_options = dict(client_options)
+        if masked_client_options.has_key("basic_auth_password"):
+            masked_client_options["basic_auth_password"] = "*****"
+        if masked_client_options.has_key("http_auth"):
+            masked_client_options["http_auth"] = (client_options["basic_auth_user"], "*****")
+        logger.info("Creating ES client connected to %s with options [%s]" % (hosts, masked_client_options))
```

Comment:
Very minor comment/ask, the current code is not consistent at the moment, but while you are at it, would it be possible to use the comma format and have the options as arguments, as documented [here](https://docs.python.org/3/library/logging.html)? The idea is that moving forwards we move away from [% for strings](https://pyformat.info/), so in your case it'd be:

```
logger.info("Creating ES client connected to %s with options [%s]", hosts, masked_client_options)
```

Thank you!

Reply:
Of course.